SPECS/mstflint/mstflint.spec

```diff
@@ -23,15 +23,35 @@ BuildRequires:  libxml2-devel
 BuildRequires:  make
 BuildRequires:  openssl-devel
 BuildRequires:  zlib-devel
+BuildRequires:  kernel-devel
```

Comment:
We're introducing an out-of-tree kernel module in this spec. This is a serious maintenance commitment to keep it up-to-date with the changing internal kernel APIs.

1. Do we have a team, who is going to make sure this module continues to work?
2. Please limit the `BuildRequires` and `Requires` to the lowest and highest kernel versions we currently know this module works with. Otherwise, we will encounter build and/or run-time issues in the future as the kernel internal APIs change and the module falls behind.

Reply:
1- Yes, Chaitanya's team. I'll be adding a test for this module in next iteration.
2- Sure.

Reply:
I looked into the way that the module is including the kernel headers, it seems the kernel header file inclusions has been handled for the older versions, so there is no point for checking the required lowest kernel version. And for the highest version check, the cost of maintenance would be more if we add the current version as the highest required version; since there is less chance that those APIs get changed on next kernel version, but that condition will definitely break the build or runtime execution. Then there is no point on limiting the highest kernel version as well. 
As I mentioned, I'm going to add a test for this module, and I think that's a better solution to ensure the module compatibility with kernel version.